eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -471,3 +471,8 @@ jobs:
       inputs:
         artifactName: docs.ms
         path: $(Build.ArtifactStagingDirectory)/docs.ms
+
+    - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+        # ComponentGovernance is currently unable to run on pull requests of public projects. Running on non-PR
```

Comment:
You should add the condition that matches this comment so that we don't try to run it on public CI runs.

Reply:
Yep, will do. I had removed it just to see what it would do and how it woul fail.